test/integration/run-command.js

```diff
@@ -30,11 +33,49 @@ function runCommand(path, args, options) {
         return;
       }
       if (code !== 0) {
-        reject(new Error('exit code: ' + code));
+        reject(new Error(
+          `Error running ${path} with args ${args}. Got exit code: ${code}`));
         return;
       }
       resolve();
```

Comment:
It would be helpful to resolve with the `pipesToString()` contents in case a consumer needs it.